env-for-circle/develop.sh

```diff
@@ -14,3 +14,5 @@ export IRS_SUPERUSER_EMAIL="${IRS_SUPERUSER_EMAIL_DEV}"
 export USTC_ADMIN_PASS="${USTC_ADMIN_PASS}"
 export USTC_ADMIN_USER="${USTC_ADMIN_USER}"
 export ZONE_NAME="${ZONE_NAME_DEV}"
+export SLACK_WEBHOOK_URL=${SLACK_WEBHOOK_URL_DEV}
```

Comment:
I think we are trying to avoid using these files.  Did you also add these to secrets manager?

Reply:
I did, and I had a question about this very file in the previous PR. It kind of appears that these files were deprecated now with the secrets solution. Thanks